ONSdigital/dp-frontend-dataset-controller
# Feature/map cookie values to page model

### What

- Move legacy mapper from `go-ns` into app
- Switch to using zebedee client response model from `dp-api-clients-go` instead of `go-ns`
- Add cookie preferences mapping to each mapper
- Update versions of `dp-api-clients-go` and `go-ns`

### How to review

- Load a page that the controller is response for (e.g. dataset landing page)
- Check that cookie preferences that are selected by a user are passed to the page model
    - easiest way is to add temporary logging to page handler in renderer
    - change cookie settings to make sure not only defaults are mapped


### Who can review

Anyone
